identity-provider-service/src/bin/identity_verifier.rs

```diff
@@ -192,6 +210,12 @@ async fn main() {
                         }
                     }
 
+                    let expiry = input
+                        .get("expiry")
+                        .map(|delay| delay.replace("-", ""))
+                        .unwrap();
```

Comment:
Can we please not unwrap and log instead? I don't like crashing if I can just reject.